app/views/shared/_footer.html.erb

```diff
@@ -14,6 +14,6 @@
 %>
 
 <footer class="footer pt-3">
-  <h5 class="text-center mb-1"><%= t("footer.powered_by", href: link_to(t("bigbluebutton"), "http://bigbluebutton.org/", target: "_blank", class: "text-blue")).html_safe %></h5>
+  <h5 class="text-center mb-1"><%= t("footer.powered_by", href: link_to(t("Greenlight"), "https://bigbluebutton.org/2018/07/09/greenlight-2-0/", target: "_blank", class: "text-green")).html_safe %></h5>
```

Comment:
Greenlight should be greenlight, and also we have to make sure that the string is localised